moveit_middleware_benchmark.repos

```diff
@@ -1,5 +1,5 @@
 repositories:
-  moveit_msgs:
+  moveit2:
```

Comment:
Do we need to build moveit2 from source or can't we use the[ available docker images](https://hub.docker.com/r/moveit/moveit2)?

Reply:
Hmmm, can be used. I will work on this one in this weekend. Thank you

Reply:
Great thanks!

Reply:
I solved it in this commits https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/16/commits/f1abbb3527b877127bd7375502a6e33e055f0881 and https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/16/commits/16601be7318a30197eda640feb91177d8c0dc37e.